### ISSUE TITLE
"Tests failed" with no indication of which tests failed

### ISSUE THREAD

**Issue by himynameisdave** [exercism/swift#524]
Hey, not sure if this is an issue with just Swift excersises, but I noticed this issue in both the **Freelancer Rates** and **Vexing Vehicle Purchase** exercises.

Whenever I run the tests and have some syntax/compiler error, it will tell me exactly what I did wrong. However (unlike in the Rust exercises or the **Layers of Lasagna** exercise), I am finding that it will not display which test cases are failing (so I have no idea how to fix them):

<img width="393" alt="image" src="https://user-images.githubusercontent.com/4298089/144149783-cf01706b-3fa7-4efe-8d53-ad5ec258a325.png">

**Comment by kotp** [comment]
Is this still happening for you on this exercise?  There was some platform wide things happening that could have caused this to happen as well.

@exercism/swift

**Comment by himynameisdave** [comment]
Yep this is still happening for both exercises. Exercises in other tracks (namely the Rust track) will display the test results correctly.

I am working in Firefox, version 95.
